apps/docs/src/guide/predicates/methods.md

```diff
@@ -0,0 +1,28 @@
+# Interacting With Predicates
+
+There are 4 ways to interact with predicates: `populateTransactionPredicateData`, `sendTransaction`, `simulateTransaction`, `getResourcesToSpend`
+
+
+## `populateTransactionPredicateData`
```

Comment:
This isn't strictly true as a `Predicate` extends an `Account` so it inherits all the account based functionality (submitting, querying etc).

Maybe this could be made more general with the subheadings:
- Checking Balances - `getBalances` & `getResourcesToSpend` (make this one tx specific)
- Transactions -`sendTransaction` / `simulateTransaction` and `populateTransactionPredicateData` (for where the predicate is added to a tx builder) 
- Transfers - `createTransfer` / `transfer`

FYI I don't necessarily agree with this design choice as there are some things in there that don't make sense i.e. signing and the use of connectors, but we should highlight the account based functionality that it does use.